nix/tests/vm-tests/marlowe-playground.nix

```diff
@@ -5,11 +5,12 @@ let
     FRONTEND_URL="http://localhost:8080"
     GITHUB_CALLBACK_PATH="/#/gh-oauth-cb"
     GITHUB_CLIENT_ID="314123123a312fe"
-    GITHUB_CLIENT_SECRET=kljfks234dskjhfeskjr"
+    GITHUB_CLIENT_SECRET="kljfks234dskjhfeskjr"
```

Comment:
how did this work before????

Reply:
This "worked" because:
- The relevant systemd bash script can fail silently (i should improve that)
- There is no obvious way to _really_ ensure/assert that the side-effect (config change) in the service really occurred.

I'll change the service code to actually make some noise when it can't read the config